clusterloader2/testing/experimental/storage/pod-startup/suite.yaml

```diff
@@ -0,0 +1,47 @@
+- identifier: emptydir-vol-per-pod
+  configPath: "testing/experimental/storage/pod-startup/config.yaml"
+  overridePaths:
+    - "./testing/experimental/storage/pod-startup/volume-types/emptydir/override.yaml"
+    - "./testing/experimental/storage/pod-startup/max_volumes_per_pod/override.yaml"
+
+- identifier: configmap-vol-per-pod
+  configPath: "testing/experimental/storage/pod-startup/config.yaml"
+  overridePaths:
+    - "./testing/experimental/storage/pod-startup/volume-types/configmap/override.yaml"
```

Comment:
Nit: I don't think you need "./" prefix.

Reply:
Fixed